dotnet/src/Connectors/Connectors.HuggingFace/Core/Models/TextEmbeddingResponse.cs

```diff
@@ -8,4 +8,5 @@ namespace Microsoft.SemanticKernel.Connectors.HuggingFace.Core;
 /// <summary>
 /// Represents the response from the Hugging Face text embedding API.
 /// </summary>
-internal sealed class TextEmbeddingResponse : List<List<List<ReadOnlyMemory<float>>>>;
+/// <returns> List&lt;ReadOnlyMemory&lt;float&gt;&gt;</returns>
+internal sealed class TextEmbeddingResponse : List<ReadOnlyMemory<float>>;
```

Comment:
This can be a breaking change for the models this service was tested, can you provide which embedding models are you testing against?

Reply:
I have used https://huggingface.co/BAAI/bge-large-en & other models as well those are mentioned   in models used  in https://github.com/microsoft/semantic-kernel/issues/6635